server/index.js

```diff
@@ -113,6 +112,46 @@ app.get("/api/eggs", (req, res, next) => {
     .catch((err) => next(err));
 });
 
+app.get("/api/profile/:userId", (req, res, next) => {
+  const userId = Number(req.params.userId);
```

Comment:
Instead of reading the user id from the params, you should use your authorization middleware's `req.user`. You can shorten the url path to just `/api/profile`